ambry-router/src/test/java/com.github.ambry.router/GetBlobOperationTest.java

```diff
@@ -94,6 +97,10 @@
   private final MockNetworkClient mockNetworkClient;
   private final RouterCallback routerCallback;
   private final String operationTrackerType;
+  private final boolean testEncryption;
+  private CryptoJobHandlerTest.MockKeyManagementService kms = null;
```

Comment:
just move these to their own file if they are used in many tests

Reply:
done.